app/controllers/jobs_controller.rb

```diff
@@ -1,44 +1,62 @@
 class JobsController < ApplicationController
+
   def index
-    @company = Company.find(params[:company_id])
-    @jobs = @company.jobs
+    @jobs = Job.all
+    return @jobs = Job.by_level_of_interest if params[:sort] == "interest"
```

Comment:
there's a lot of logic here - could extract this out into another method possibly to keep this method more DRY?